packages/shared/warningWithoutStack.js

```diff
@@ -27,7 +29,14 @@ if (__DEV__) {
     }
     if (typeof console !== 'undefined') {
       const stringArgs = args.map(item => '' + item);
+      const originalConsoleError = console.error;
+
+      if (typeof console.error.apply === 'undefined') {
+        console.error = consoleError;
```

Comment:
There are more moving parts here than I'd like.

I'd like to use the same solution for all cases. Can we do that somehow?

The easiest one might just be to switch on `args` length and explicitly pass them all down.

Reply:
That was my first intention :)
But then I thought that maybe it’s not desired to switch to fixed amount of args.
If it’s ok for you - I can update.

Reply:
Fixed amount of args is okay. We have that in `invariant`.